src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -97,4 +97,8 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     val key = SymbolStoreKey(scope, name, iteration = None, input = false)
     store.find(_.key == key)
   }
+
+  def getOutputs: Set[SymbolStoreEntry] = {
+    store filter {e => !e.isInput} toSet
```

Comment:
There should probably be an `isOutput` method, and this should call it.  :smile: